src/index.js

```diff
@@ -10,25 +10,33 @@ const artistList = document.getElementById('artist-list');
 
 artistList.addEventListener('click', async (e) => {
   const artistId = e.target.parentElement.parentElement.id;
+  commentPopup.innerHTML = '';
```

Comment:
**[OPTIONAL]**: 

- [ ] Please consider using a different approach since `.innerHTML` can be easily modified and can be harmful :warning:  for the app.